generators/beamAndVertexGenerator.cc

```diff
@@ -235,3 +471,27 @@ ostream& beamAndVertexGenerator::print(ostream& out) const
 	return out;
 
 }
+
+namespace{
+TVectorD multivariateGauss(const TVectorD& mean, const TMatrixDSym& cov, const double sigmaScalingFactor)
+{
+	TRandom3* randomGen = randomNumberGenerator::instance()->getGenerator();
+	const int ndim = mean.GetNrows();
+	if ( ndim != cov.GetNrows() or ndim != cov.GetNcols()){
+		printErr << "Covariance matrix or mean vector have wrong format" << endl;
+		throw "Covariance matrix or mean vector have wrong format";
```

Comment:
Is the doubling of the error message needed?

Reply:
Yes, to have it in the exception.
This is useful if the exception is catched by Python!